### ISSUE TITLE
Where can I get the files in the path？

### ISSUE THREAD

**Issue by oula-le** [anderzzz/monkey_caput#3]
Hello, I want to reproduce the code in this paper, but I encountered a problem in the process, so I want to ask you for advice. I want to ask  where are the files in the file path in "raw_csv_toc='../../Desktop/Fungi/toc_full.csv', raw_csv_root='../../Desktop/Fungi"In line 32 of ae_runs.py? Thank you for your time to look this and I look forward to your reply.

**Comment by anderzzz** [comment]
Hello. That's the path to the raw data for training and testing. It is not included in the repo. You can replace that path with a folder to your own image data.

You can read my detailed reply to a similar question here: https://github.com/anderzzz/monkey_caput/issues/1#issuecomment-814136443

As you see I am not able to provide the dataset, since I scraped the images and I have not been able to received copyright approval by the database owner. Note however that part of the dataset has been made public before, so maybe you can convince the database owners (I couldn't)? Or try with your own image data.

**Comment by oula-le** [comment]
Thank you very much for your answer. I will try to change it into my own training set and test set of picture data.
